intro/notebooks-intro/notebooks-intro.ipynb

```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "1. Edit the following markdown cell with the *CHANGE_ME* text.\n",
```

Comment:
The indentation in this step is a bit strange
![image](https://github.com/RedHatTraining/AI26X-apps/assets/675470/20a2855b-47af-4cf1-a9b8-eae76f017569)

We can fix it if we just add a tab in the sentences under the numbered step.
![image](https://github.com/RedHatTraining/AI26X-apps/assets/675470/88aed159-6d53-4adb-9d88-65c7d8ecd577)

![image](https://github.com/RedHatTraining/AI26X-apps/assets/675470/7e2e06d6-2383-47fb-8856-37f1919f8b5d)